apps/shelter-client/src/libs/Chat.tsx

```diff
@@ -1,16 +1,22 @@
 import '../styles/Chat.scss'
-import { FC, useEffect, useRef, useState } from 'react';
+import React, { FC, useEffect, useRef, useState } from 'react';
```

Comment:
Seems that this approach brings in the entire React library, including all its functionalities. Let's make like that
**import {MouseEvent} from 'react'**
to reduce bundle sizes.